assets/js/actions/apollo.js

```diff
@@ -102,27 +103,20 @@ export const setupApolloClient = (getAuthToken, organizationId) => {
     socket.connect()
 
     store.subscribe(async () => {
-      const newTokenClaims = await getAuthToken();
-      const newToken = newTokenClaims.__raw;
-      const newOrganization = store.getState().organization;
-
-      if (newToken !== token || currentOrganizationId !== newOrganization.currentOrganizationId) {
-        currentOrganizationId = newOrganizationId
-        token = newToken;
-        socket.disconnect();
-        socket = createSocket(token, currentOrganizationId)
-        socket.connect()
+      if (Math.ceil(Date.now() / 1000) > store.getState().apollo.tokenClaims.exp) {
+        logout()
```

Comment:
could we alert user before this happens and give them an opportunity to renew the session?

Reply:
i still havent been able to figure out token refresh, so probably try to make that in a separate PR. In the meantime, I'll make the token expiration on prod like 36 hours or something

Reply:
the docs are terrible

Reply:
for sure, we can address the immediate urgency of logging user out and tackle refreshing session in future ticket to improve UX